src/script/buildcontainer-setup.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
can we just use the steps at https://docs.ceph.com/en/squid/start/beginners-guide/ to setup the environment? 

as in use `install-deps.sh` so that we can keep all the dependencies per OS etc are all in the same place?

Reply:
There's a maze of twisty passages throughout these build scripts. This eventually does end up calling install-deps.sh. Without doing a deep check the flow is "include run-make.sh" -> prepare -> install-deps.sh.

I'd love to simplify some of this but this script was added to (a) match more closely what the jenkins builds are doing (b) add extra packages needed to "fix" the container environment where install-deps either misses or doesn't need something but the containerized env does.

Reply:
gotcha.

so is the goal - to deprecate `install-deps.sh` once we get this simplified version done? so that how folks build it locally and how its build in jenkins is consistent(?)

As for the "additional" dependencies needed just for container builds - would it make sense to keep that as a separate list/script so that we can dig into why that difference exists and document it out?